### PR TITLE
Fix mdxcreateelement

### PR DESCRIPTION
**Pull request by haphizi**

### CODE REVIEW

**Comment by haphizi** [comment]
fyi guys, for variants that are using external jsx code snippet, i disabled the canvas code functionality. reflects 'code not available'

File: stories/Combobox/Combobox.stories.mdx [hunk 1]
```diff
@@ -2,10 +2,7 @@ import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
 import { Combobox, Form } from '../../src/index';
```
Comment (clukhei) [inline stories/Combobox/Combobox.stories.mdx:2]: Like Dropdown, Combobox also have menu up and down. Can revert this file to the usual one , dont need to use canvas and just show normal snippet is fine..

File: stories/FileUpload/FileUpload.stories.mdx [hunk 2]
```diff
@@ -5,10 +5,55 @@ import { FileUpload } from '../../src/FileUpload/FileUpload';
 import { Form } from '../../src/Form/index';
 import { HIDDEN_COMMON_PROPS } from '../helpers';
 
-<Meta
-  title="Components/File Upload"
-  component={FileUpload}
-/>
+<Meta title="Components/File Upload" component={FileUpload} />
+
+export const templateCode_FileUpload = `
```
Comment (clukhei) [inline stories/FileUpload/FileUpload.stories.mdx:10]: keep variable name to camelCase, dont need underscore _

File: stories/Footer/Footer.stories.mdx [hunk 3]
```diff
@@ -18,6 +18,147 @@ import { HIDDEN_COMMON_PROPS } from '../helpers';
   }}
 />
 
+export const templateCode_Footer = `
```
Comment (clukhei) [inline stories/Footer/Footer.stories.mdx:21]: keep variable names to camelCase

**Review by sophiaaachow — APPROVED** [review]

**Review by clukhei — APPROVED** [review]

**Pull request merged**